src/esn.inbox.libs/app/services/mailboxes/mailboxes-service.js

```diff
@@ -88,14 +88,16 @@ angular.module('esn.inbox.libs')
 
         if (item.parentId) {
           const parentItem = arrMap.get(item.parentId);
-          const { nodes } = parentItem;
+          const nodes = parentItem.nodes;
 
-          if (parentItem && parentItem.id !== item.id && !_.find(nodes, { id: item.id })) {
+          if (parentItem && parentItem !== null && parentItem.id !== item.id && !_.find(nodes, { id: item.id })) {
```

Comment:
`parentItem !== null` condition would always be true here, I don't understand why it was added.

Reply:
The mailbox could have a parent mailbox not in this list that why the is a else (for example the case of the parent is a special mailbox so not in this list).

Reply:
I mean, checking `&&` conditions :
`parentItem` it must be truish
`parentItem !== null` must not be null, but null is falsish so it is already checked by previous expression isn't it?
`parentItem.id !== item.id` must not have same id than item
`!_.find(nodes, { id: item.id })` must not have nodes with same id than item